code/__HELPERS/traits.dm

```diff
@@ -202,6 +202,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_UNREVIVABLE 		"unrevivable" // Prevents changeling revival
 #define TRAIT_CULT_IMMUNITY		"cult_immunity"
 #define TRAIT_FLATTENED			"flattened"
+#define TRAIT_RESPAWNABLE		"respawnable" // Allows respawnable people to
```

Comment:
Honestly, I feel like you might want to make this change in how respawnability works its own atomic PR. It's a big thing on its own.